qtt/instrument_drivers/virtualAwg/templates.py

```diff
@@ -60,3 +60,7 @@ def marker(name):
         """
         return TablePT({name: [(0, 0), ('offset', 1),
                                ('offset+uptime', 0), ('period', 0)]})
+
+    @staticmethod
+    def pulse_table(name, entries):
```

Comment:
Missing docstring. The function is so short, do we need it?

Reply:
✔️